BeVolt/App/src/CLI.c

```diff
@@ -0,0 +1,723 @@
+/* CLI.c
+ * Command Line Interface wrapper to 
+ * define and route commands
+ */
+
+#include "CLI.h"
+#include <ctype.h>
+#include <string.h>
+#include "stm32f4xx.h"
+#include "stm32f4xx_gpio.h"
+
+#define MAX_TOKEN_SIZE 4
+
+static cell_asic* Minions;
+int32_t hashTokens[MAX_TOKEN_SIZE];

```

Comment:
What advantage does having a global array that you use for parameter passing give you over simply having relevant arguments for each command that you pass to their functions?

Reply:
I'm not sure what explicitly I had in mind in terms of advantages when I did that, but I'll change it to local variable in the handler that gets passed to the functions. I believe I originally tried something like that and ran into errors with the string implementation, but with the hashed implementation it should be simple.